src/app/docs/datepicker/datepicker-docs.component.html

```diff
@@ -7,14 +7,107 @@
     The datepicker module provides an input and calendar for users to select dates or fuzzy dates.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-code-examples
+  <sky-docs-demo
+    [heading]="'Standard datepicker demo'"
+  >
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include date range validator (01/01/2020 - 01/31/2020)"
+          propertyName="standardValidation"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <div
+      style="width:400px;"
+    >
+      <form
+        novalidate
+        [formGroup]="standardForm"
+      >
+        <sky-datepicker>
```

Comment:
Should the datepickers in the demo have labels? I only noticed because I was checking to see if we needed to include an ARIA label in the code sample for accessibility, and I realized that we don't have an ARIA label for this component. Should we create an issue for that? And in the shorter term, would associating the datepickers in the demo help at all from an accessibility-example viewpoint?

Reply:
Good thought. I don't think an ARIA input would apply here because the consumer has control over the actual HTML `input` element. I've updated the demo to show how they would apply a proper label. That should cover us.